src/components/EarnDashboard/components/ProviderDetails/LpPositionsByPosition.tsx

```diff
@@ -0,0 +1,139 @@
+import { Button, Flex } from '@chakra-ui/react'
+import { Tag } from '@chakra-ui/tag'
+import { fromAssetId } from '@shapeshiftoss/caip'
+import { DefiAction } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import qs from 'qs'
+import { useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory, useLocation } from 'react-router'
+import type { Column, Row } from 'react-table'
+import { Amount } from 'components/Amount/Amount'
+import { ReactTable } from 'components/ReactTable/ReactTable'
+import { AssetCell } from 'components/StakingVaults/Cells'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { trackOpportunityEvent } from 'lib/mixpanel/helpers'
+import { MixPanelEvents } from 'lib/mixpanel/types'
+import type { LpEarnOpportunityType, OpportunityId } from 'state/slices/opportunitiesSlice/types'
+import { selectAggregatedEarnUserLpOpportunities, selectAssets } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+type LpPositionsByPositionProps = {
+  ids: OpportunityId[]
+}
+
+export type RowProps = Row<LpEarnOpportunityType>
+
+export const LpPositionsByPosition: React.FC<LpPositionsByPositionProps> = ({ ids }) => {
+  const translate = useTranslate()
+  const location = useLocation()
+  const history = useHistory()
+  const {
+    state: { isConnected, isDemoWallet },
+    dispatch,
+  } = useWallet()
+  const assets = useAppSelector(selectAssets)
+  const lpOpportunities = useAppSelector(selectAggregatedEarnUserLpOpportunities)
+  const filteredDown = lpOpportunities.filter(e => ids.includes(e.assetId as OpportunityId))
```

Comment:
This was only moved around but if we want to give the callback item a better name might as well